{
    "containers": {
        "cna": {
            "affected": [
                {
                    "product": "n/a",
                    "vendor": "n/a",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "n/a"
                        }
                    ]
                }
            ],
            "datePublic": "2004-03-09T00:00:00",
            "descriptions": [
                {
                    "lang": "en",
                    "value": "Buffer overflow in the getaddrinfo function in Python 2.2 before 2.2.2, when IPv6 support is disabled, allows remote attackers to execute arbitrary code via an IPv6 address that is obtained using DNS."
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "description": "n/a",
                            "lang": "en",
                            "type": "text"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "dateUpdated": "2004-08-18T00:00:00",
                "orgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
                "shortName": "mitre"
            },
            "references": [
                {
                    "name": "9836",
                    "tags": [
                        "vdb-entry",
                        "x_refsource_BID"
                    ],
                    "url": "http://www.securityfocus.com/bid/9836"
                },
                {
                    "name": "MDKSA-2004:019",
                    "tags": [
                        "vendor-advisory",
                        "x_refsource_MANDRAKE"
                    ],
                    "url": "http://www.mandrakesoft.com/security/advisories?name=MDKSA-2004:019"
                },
                {
                    "name": "4172",
                    "tags": [
                        "vdb-entry",
                        "x_refsource_OSVDB"
                    ],
                    "url": "http://www.osvdb.org/4172"
                },
                {
                    "name": "GLSA-200409-03",
                    "tags": [
                        "vendor-advisory",
                        "x_refsource_GENTOO"
                    ],
                    "url": "http://www.gentoo.org/security/en/glsa/glsa-200409-03.xml"
                },
                {
                    "name": "DSA-458",
                    "tags": [
                        "vendor-advisory",
                        "x_refsource_DEBIAN"
                    ],
                    "url": "http://www.debian.org/security/2004/dsa-458"
                },
                {
                    "name": "python-getaddrinfo-bo(15409)",
                    "tags": [
                        "vdb-entry",
                        "x_refsource_XF"
                    ],
                    "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/15409"
                }
            ],
            "x_legacyV4Record": {
                "CVE_data_meta": {
                    "ASSIGNER": "cve@mitre.org",
                    "ID": "CVE-2004-0150",
                    "STATE": "PUBLIC"
                },
                "affects": {
                    "vendor": {
                        "vendor_data": [
                            {
                                "product": {
                                    "product_data": [
                                        {
                                            "product_name": "n/a",
                                            "version": {
                                                "version_data": [
                                                    {
                                                        "version_value": "n/a"
                                                    }
                                                ]
                                            }
                                        }
                                    ]
                                },
                                "vendor_name": "n/a"
                            }
                        ]
                    }
                },
                "data_format": "MITRE",
                "data_type": "CVE",
                "data_version": "4.0",
                "description": {
                    "description_data": [
                        {
                            "lang": "eng",
                            "value": "Buffer overflow in the getaddrinfo function in Python 2.2 before 2.2.2, when IPv6 support is disabled, allows remote attackers to execute arbitrary code via an IPv6 address that is obtained using DNS."
                        }
                    ]
                },
                "problemtype": {
                    "problemtype_data": [
                        {
                            "description": [
                                {
                                    "lang": "eng",
                                    "value": "n/a"
                                }
                            ]
                        }
                    ]
                },
                "references": {
                    "reference_data": [
                        {
                            "name": "9836",
                            "refsource": "BID",
                            "url": "http://www.securityfocus.com/bid/9836"
                        },
                        {
                            "name": "MDKSA-2004:019",
                            "refsource": "MANDRAKE",
                            "url": "http://www.mandrakesoft.com/security/advisories?name=MDKSA-2004:019"
                        },
                        {
                            "name": "4172",
                            "refsource": "OSVDB",
                            "url": "http://www.osvdb.org/4172"
                        },
                        {
                            "name": "GLSA-200409-03",
                            "refsource": "GENTOO",
                            "url": "http://www.gentoo.org/security/en/glsa/glsa-200409-03.xml"
                        },
                        {
                            "name": "DSA-458",
                            "refsource": "DEBIAN",
                            "url": "http://www.debian.org/security/2004/dsa-458"
                        },
                        {
                            "name": "python-getaddrinfo-bo(15409)",
                            "refsource": "XF",
                            "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/15409"
                        }
                    ]
                }
            }
        }
    },
    "cveMetadata": {
        "assignerOrgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
        "assignerShortName": "mitre",
        "cveId": "CVE-2004-0150",
        "datePublished": "2004-09-01T04:00:00",
        "dateReserved": "2004-02-13T00:00:00",
        "dateUpdated": "2004-08-18T00:00:00",
        "state": "PUBLISHED"
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.0"
}